backend/web/index.php

```diff
@@ -1,7 +1,4 @@
 <?php
-defined('YII_DEBUG') or define('YII_DEBUG', true);
-defined('YII_ENV') or define('YII_ENV', 'dev');
```

Comment:
Can't do it since after `require(__DIR__ . '/../../vendor/yiisoft/yii2/Yii.php');` setting debug mode won't have any effect: `define('YII_DEBUG', true)` won't work.